tests/test_layout.py

```diff
@@ -369,23 +369,23 @@ def test_cycle_stays_in_floating_layer(hlwm):
     for winid in floating:
         assert hlwm.attr.clients.focus.winid() == winid
         # the floating layer always stays focused
-        assert hlwm.attr.tags.focus.floating_focused() == hlwm.bool(True)
+        assert hlwm.attr.tags.focus.floating_focused() is True
```

Comment:
```suggestion
        assert hlwm.attr.tags.focus.floating_focused()
```

maybe? We're not really testing the `floating_focused` getter type here, and I assume in C(++) you'd also write `if (floating_focused)` rather than `if (floating_focused == true)`.

(Same for all other instances of `is True` and `is False`, except maybe `test_implicit_type_conversion_bool` because we're really interested in the type there)

Reply:
Hm you're probably right, but it feels fragile because the type is detected at run-time and returning the string `"false"` here would make the `assert` hold. (Also, I think `assert … is True` reads quite well)